gnosis/eth/tests/test_ethereum_client.py

```diff
@@ -765,7 +765,7 @@ def test_trace_filter(self):
             ValueError, "The method trace_filter does not exist/is not available"
         ):
             self.ethereum_client.parity.trace_filter(
-                from_address=Account.create().address
+                to_address=[Account.create().address]
```

Comment:
I didn't understand this change, it was filtered first for from_address and now for to_address. It is correct?

Reply:
The method is supposed to wait for a list of addresses, we had it wrong: https://openethereum.github.io/JSONRPC-trace-module#trace_filter